src/System.Private.Xml/src/System/Xml/XmlDownloadManager.cs

```diff
@@ -2,25 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Cache;
+using System.Net.Http;
+
 namespace System.Xml
 {
-    using System;
-    using System.IO;
-    using System.Security;
-    using System.Collections;
-    using System.Net;
-    using System.Net.Cache;
-    using System.Runtime.CompilerServices;
-    using System.Runtime.Versioning;
-    using System.Net.Http;
-
-    //
-    // XmlDownloadManager
-    //
     internal partial class XmlDownloadManager
     {
-        internal Stream GetStream(Uri uri, ICredentials credentials, IWebProxy proxy,
-            RequestCachePolicy cachePolicy)
+        internal Stream GetStream(Uri uri, ICredentials credentials, IWebProxy proxy, RequestCachePolicy cachePolicy)
```

Comment:
Seems like 'cachePolicy' isn't being used anymore. And it doesn't really do anything on .NET Core anyways.

Reply:
Yup.  Removed.